apps/innsendt-soeknad/src/main/kotlin/person/krr/KrrKlient.kt

```diff
@@ -31,16 +34,16 @@ class KrrKlient(
 
         return try {
             val response =
-                client.get("person") {
-                    header(HttpHeaders.NavPersonIdent, fnr.value)
+                client.post("personer") {
                     header(HttpHeaders.NavCallId, UUID.randomUUID().toString())
                     header(X_CORRELATION_ID, getCorrelationId())
                     accept(ContentType.Application.Json)
                     contentType(ContentType.Application.Json)
+                    setBody(DigitalKontaktinformasjonRequestBody(listOf(fnr.value)))
                 }
 
             if (response.status.isSuccess()) {
-                response.body()
+                response.body<DigitalKontaktinformasjonResponseBody>().personer[fnr.value]
```

Comment:
En liten headsup! Nytt endepunkt gir alltid 200 OK (med noen unntak selvsagt), så du må manuelt håndtere feilene. 
Response-objektet kan inneholde `personer` og/eller `feil`. 

Se her: 
https://github.com/navikt/digdir-krr/wiki/Migrere-vekk-fra-GET%E2%80%90tjenesten-for-enkeltoppslag

Ikke veldig farlig siden data fra KRR ikke er kritisk i løsningen, men greit å være obs på.

Reply:
Thank you `<BodyShort />`. I will keep in mind